instrumentation/servlet/glassfish-testing/src/test/java/TestServlets.java

```diff
@@ -20,14 +19,11 @@ protected void service(HttpServletRequest req, final HttpServletResponse resp) {
           HttpServerTest.ServerEndpoint.forPath(req.getServletPath());
       HttpServerTest.controller(
           endpoint,
-          new Callable<Object>() {
-            @Override
-            public Object call() throws Exception {
-              resp.setContentType("text/plain");
-              resp.setStatus(endpoint.getStatus());
-              resp.getWriter().print(endpoint.getBody());
-              return null;
-            }
+          () -> {
```

Comment:
I remember trying this and reverting with a failure since it seemed like the lambda couldn't be instrumented. Maybe misunderstood though

Reply:
oh yes, CI backed up your story 😄

found this: https://www.adam-bien.com/roller/abien/entry/glassfish_v4_0_1_daily

bumping from glassfish 4.0 to 4.1 (no 4.0.1 in maven) seems to resolve